stdlib/nodejs/pathname.rb

```diff
@@ -0,0 +1,30 @@
+class Pathname
+
+  @__fs__ = node_require :fs
+  `var __fs__ = #{@__fs__}`
+
+  # Instance Methods
+
+  def initialize(path)
+    #raise ArgumentError if path == "\0"
+    if Pathname === path
+      @path = path.path
+    else
+      @path = path
+    end
+  end
```

Comment:
@elia but `stdlib/nodejs/pathname` still need a constructor no ? I can safely remove `initialize` from `stdlib/nodejs/pathname` and Opal will use `initialize` method from `stdlib/pathname` instead ?